src/phpDocumentor/DomainModel/Uri.php

```diff
@@ -98,7 +98,10 @@ private function validateUri($uri)
      */
     private function addFileSchemeWhenSchemeIsAbsent($uri): string
     {
-        if (!parse_url($uri, PHP_URL_SCHEME)) {
+        $results = parse_url($uri, PHP_URL_SCHEME);
+        if ($results === 'C') {
```

Comment:
Think this is to limited. C is the default drive on windows. But it can be any letter. Think the check should be something like  `preg_match($results, '[a-z]')`